renderer/components/editor/options/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Size from './size';
+import Fps from './fps';
+import ExportSelect from './export-select';
+
+const TEST_OPTIONS = [{label: 'Export to disk', value: 'test'}];
```

Comment:
Leftover?

Reply:
It's there to show the dropdown, should be removed when it's hooked up by @karaggeorge, but I can remove it now if you'd like?